### PR TITLE
Response card improvements

### PR DESCRIPTION
**Pull request by DavDo**
MOB-1566

**Jira issue:**
https://glia.atlassian.net/browse/MOB-1566

**Additional info:**

Response cards' behavior on mobile will be unified with the behavior on web as follows:
- Upon clicking on the card, the card disappears and is replaced with the visitor's response as a text 
- Response cards are not shown in chat history

[MOB-1566]: https://glia.atlassian.net/browse/MOB-1566?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MOB-1566]: https://glia.atlassian.net/browse/MOB-1566?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gugalo — APPROVED** [review]

**Review by AndriiHorishniiMOC — APPROVED** [review]

File: version.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #Wed Dec 07 09:39:06 UTC 2022
-dependency.coreSdk.version=0.26.5
+dependency.coreSdk.version=0.27.0
```
Comment (andrews-moc) [inline version.properties:2]: `0.27.0` doesn't exist yet, does it?

Comment (DavDo) [reply version.properties:2]: it shouldn’t be there, I changed it for test purposes and forgot to change it back.

Comment (DavDo) [reply version.properties]: fixed [81eeee9](https://github.com/salemove/android-sdk-widgets/pull/418/commits/81eeee9b0c65adf94e0f404c3c89a4ec23414ef7)

File: widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java [hunk 2]
```diff
@@ -1412,6 +1409,9 @@ public void newEngagementLoaded(OmnicoreEngagement engagement) {
             Logger.d(TAG, "unsentMessage sent!");
         }
         emitViewState(chatState.engagementStarted());
+        // Loading chat history again on engagement start in case it was an-authenticated visitor that restored ongoing engagement
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java:1412]: an-authenticated => un-authenticated

File: widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java [hunk 3]
```diff
@@ -1412,6 +1409,9 @@ public void newEngagementLoaded(OmnicoreEngagement engagement) {
...
+        // Currently there is no direct way to know if Visitor is authenticated.
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java:1413]: We can use `authentication.isAuthenticated()` to know if Visitor is authenticated.

File: widgetssdk/src/main/java/com/glia/widgets/chat/model/history/LinkedChatItem.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    }
+
+    public String getMessageId() {
+        return messageId;
+    }
+
+    public long getTimestamp() {
+        return timestamp;
+    }
+}
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/chat/model/history/LinkedChatItem.java:21]: Hm, I can't understand why `LinkedChatItem` is shown as a change in this PR, taking into account that `LinkedChatItem` is already added to the target branch `feature/MOB-1612_chat_history`. Am I missing anything?

**Comment by DavDo** [comment]
moved to #423

**Pull request closed without merging**